index.ts

```diff
@@ -33,18 +34,19 @@ class ElectronForgeSslCodeSignPlugin extends PluginBase<ConfigTypes> {
 		forgeConfig: ResolvedForgeConfig,
 		makeResults: ForgeMakeResult[],
 	) => {
-		const squirrelMaker: IForgeResolvableMaker = forgeConfig.makers.find(
-			(maker) =>
-				(maker as IForgeResolvableMaker)?.name ===
-				"@electron-forge/maker-squirrel",
-		) as IForgeResolvableMaker
+		const _squirrelMaker = forgeConfig.makers.find(
+			(maker) => "name" in maker && !maker.name.includes("squirrel"),
```

Comment:
There is probably a typo here, I assume `!maker.name.includes("squirrel")` should be `maker.name.includes("squirrel")`.